src/webview/columns/data-column.tsx

```diff
@@ -82,13 +84,14 @@ export class EditableDataColumnRow extends React.Component<EditableDataColumnRow
     }
 
     protected renderGroup(maus: React.ReactNode, startAddress: bigint, endAddress: bigint): React.ReactNode {
+        const breakpointMetadata = breakpointService.metadata(toHexStringWithRadixMarker(startAddress));
         return <span
-            className='byte-group hoverable'
+            className={classNames('byte-group', 'hoverable', ...BreakpointService.inlineClasses(breakpointMetadata))}
```

Comment:
This looks like it should perhaps be done as decorations rather than class names.